RunningOrder/Story/Views/StoryList+Logic.swift

```diff
@@ -42,5 +50,40 @@ extension StoryList {
 
             return Color.Identifier.epicColors[epicIndex]
         }
+
+        var allStories: [Story] {
+            var wholeStories: [Story] = []
+            sprintManager.sprints.forEach { sprint in
```

Comment:
we shouldn't access to all the existing stories in this logic object as we only display these for the current sprint (and furthermore these for the current Space !)